wgsl/index.bs

```diff
@@ -6463,8 +6461,6 @@ Then, for example, integers 2<sup>28</sup> and 1+2<sup>28</sup> both map to the
 least significant 1 bit is not representable by the floating point format.
 This kind of collision occurs for pairs of adjacent integers with a magnitude of at least 2<sup>25</sup>.
 
-Issue: (dneto) Default rounding mode is an implementation choice.  Is that what we want?
```

Comment:
Right, we should delete this now, because we've agreed there is no specified default rounding mode.